src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -982,21 +1186,116 @@ def _update_velocity_stopping(self, summed_field_values):
 
         self._stop_particles(particles_to_be_stopped_mask)
 
+    def _update_velocity_scattering(self):
+        speeds = np.linalg.norm(self.v[self._tracked_particle_mask], axis=-1)
+
+        match self._scattering_method:
+            # Numerically integrate the scattering integral to construct the mean square scatter rate
+            case "differential cross section":
+                # `theta_prime` dummy variable introduced to prevent global shadow of `theta`
+                def mean_squared_integrand(theta_prime, speed, T_e, n_e):
+                    return (
+                        theta_prime**2
+                        * self._scattering_routine(
+                            theta_prime,
+                            speed,
+                            self._particle,
+                            self._target,
+                            T_e,
+                            n_e,
+                        )
+                    ).si.value
+
+                # Calculate the minimum scattering angle based on screening
```

Comment:
Write out all of what happens in both methods and how that is applied to the particles in latex math in the docstring for adding scattering. Bonus, you can then put that in the paper!

Reply:
Sounds good. Did you have any recommended resources for scattering? I'm reading through Taylor's Classical Mechanics at the moment

Reply:
I'll send you some stuff! I mostly meant just the equations showing how the mean scattering angle is obtained from the dsigma/domega, and provide references for the ENDF data vs. the analytic model (provide an equation for the analytic model). Basically just nicely write out what's in the code in more readable latex and explain where it comes from.